src/services/item/plugins/recycled/test/ws.test.ts

```diff
@@ -164,9 +164,14 @@ describe('Recycle websocket hooks', () => {
       if (!updatedParent) throw new Error('item should be found in test');
 
       await waitForExpect(() => {
-        const [selfDelete, childDelete] = itemUpdates;
-        expect(selfDelete).toMatchObject(SelfItemEvent('delete', updatedParent));
-        expect(childDelete).toMatchObject(ChildItemEvent('delete', updatedChild));
+        // todo: this part of the test seemed flaky as the order of the updates may not be guarantied
+        // const [selfDelete, childDelete] = itemUpdates;
+        expect(itemUpdates.find((v) => v.kind === 'self')).toMatchObject(
+          SelfItemEvent('delete', updatedParent),
+        );
+        expect(itemUpdates.find((v) => v.kind === 'child')).toMatchObject(
+          ChildItemEvent('delete', updatedChild),
+        );
```

Comment:
Why do you need to change that? Seems legit but I don't get it.

Reply:
So this is not directly related to the issue with the badWords.
This test has been flaky in the CI for some time. It looks like there is supposed to be two updates that are received but they may not arrive in the expected order. So the test fails because it gets the second one before the first one. This is just to fix that flakyness.